CHANGELOG.md

```diff
@@ -1,6 +1,60 @@
 # Changelog
 
-## v5.3.0- 2025-04-29
+## v6.0.0 - 2025-05-07
+
+### Highlights
+
+- **Breaking Change**: Changed the core crypto database key format, to enable validation of the same and ensure consistency between platforms
+- Added a function for each platform to migrate from the old to the new key type
+- **Caution**: Instantiating this version of core crypto will fail before you called the migration function
+
+### Bug Fixes
```

Comment:
We're missing a `Breaking changes` section here. For every breaking change we should list affected platforms and provide advice on how to migrate. Please see how it was done for 4.0.0.